<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REDTabSettingsServerMonitoring</class>
 <widget class="QWidget" name="REDTabSettingsServerMonitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QScrollArea" name="sarea_sm">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>846</width>
        <height>623</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pbutton_sm_refresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pbutton_sm_save">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="gbox_sm_hosts">
         <property name="title">
          <string>Host Configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="2">
           <widget class="QTreeView" name="tview_sm_hosts"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbutton_sm_add_host">
            <property name="text">
             <string>Add Host</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Remove All Hosts</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="gbox_sm_email">
         <property name="title">
          <string>Email Configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_sm_email_from">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="ledit_sm_email_to"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSpinBox" name="sbox_sm_email_port">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_sm_email_to">
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_sm_email_server">
            <property name="text">
             <string>Server:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_sm_email_username">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="ledit_sm_email_from"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="chkbox_sm_email_enable">
            <property name="text">
             <string>Enable E-mail Notifications</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_sm_email_password">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_sm_email_port">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="ledit_sm_email_password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="ledit_sm_email_server"/>
          </item>
          <item row="6" column="2">
           <widget class="QCheckBox" name="chkbox_sm_email_password_show">
            <property name="text">
             <string>Show Characters</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="ledit_sm_email_username"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_sm_email_tls">
            <property name="text">
             <string>SSL/TLS:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QCheckBox" name="chkbox_sm_email_tls">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="gbox_sm_rules">
         <property name="title">
          <string>Rule Configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QPushButton" name="pbutton_sm_remove_all_rules">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove All Rules</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTreeView" name="tview_sm_rules">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbutton_sm_add_rule">
            <property name="text">
             <string>Add Rules</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>tview_sm_rules</zorder>
         <zorder>pbutton_sm_add_rule</zorder>
         <zorder>pbutton_sm_remove_all_rules</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_sm_working_wait">
     <property name="text">
      <string>Working, please wait...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_sm_disabled">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Server monitoring service is disabled.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_sm_unsupported">
     <property name="text">
      <string>Server monitoring service requires image version &gt;= 1.6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="pbar_sm_working_wait">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sarea_sm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
